<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainUI</class>
 <widget class="QWidget" name="MainUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OlympicScoreboard</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/Olympic.ico</normaloff>:/image/Olympic.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 194, 122)</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>660</width>
     <height>360</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(221, 221, 221);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Initialize">
    <property name="accessibleName">
     <string/>
    </property>
    <attribute name="title">
     <string>成绩录入</string>
    </attribute>
    <widget class="QPushButton" name="viewResultsButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>查看存储数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>80</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>提交</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="countryLineEdit">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>190</y>
       <width>300</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="countryLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>160</y>
       <width>250</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请按名次顺序输入对应国家编号(以空格分割)：&quot;</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="topThreeRadioButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>前三名</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="topFiveRadioButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>前五名</string>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLogicLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>130</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择评分逻辑：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="projectLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="projectLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>100</y>
       <width>150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入项目编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="instructionLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>请输入项目编号、选择评分逻辑，并输入对应名次的国家编号：</string>
     </property>
    </widget>
    <widget class="QLabel" name="infoLabel">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>10</y>
       <width>300</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="imageLabel">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>15</y>
       <width>250</width>
       <height>300</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#imageLabel
{
	border-image: url(:/image/Page1_intro.png);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SortPart">
    <property name="styleSheet">
     <string notr="true">QWidget#SortPart
{
background-color: rgb(221, 221, 221);
background-image: url(:/image/MedalsApparent.png);
background-position: center;
background-size: 80%;
}</string>
    </property>
    <attribute name="title">
     <string>奖牌榜</string>
    </attribute>
    <widget class="QComboBox" name="SelectBy">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox#SelectBy {
    background-color: white;
}
QComboBox#SelectBy QAbstractItemView {
    background-color: white;
}

</string>
     </property>
     <item>
      <property name="text">
       <string>国家编号</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>国家团体总分</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>国家男子项目总分</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>国家女子项目总分</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="SelectionHint">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#SelectionHint
{
background-color: white;
font: 12pt &quot;Microsoft YaHei UI&quot;;
}</string>
     </property>
     <property name="text">
      <string>  请选择奖牌榜排序依据：</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ScoreBoard">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>81</y>
       <width>421</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QTableWidget#ScoreBoard
{
	background-color: rgb(255, 255, 255);
     gridline-color: black;
}
QHeaderView::section {
    background-color: lightgray;
}</string>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <attribute name="horizontalHeaderDefaultSectionSize">
      <number>100</number>
     </attribute>
     <column>
      <property name="text">
       <string>排序名次</string>
      </property>
      <property name="background">
       <color>
        <red>213</red>
        <green>195</green>
        <blue>121</blue>
       </color>
      </property>
     </column>
     <column>
      <property name="text">
       <string>国家编号与对应总分</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QWidget" name="DetailQuery">
    <property name="styleSheet">
     <string notr="true">
QWidget#DetailQuery
{
	border-image: url(:/image/projects.png);
	background-color: rgb(221, 221, 221);
}
</string>
    </property>
    <attribute name="title">
     <string>详情查询</string>
    </attribute>
    <widget class="QFrame" name="CountryDetail">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>40</y>
       <width>250</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#CountryDetail
{
	background-color: rgba(213, 195, 121, 210)
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="CountryDetailHint">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#CountryDetailHint
{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>  按照国家编号查询：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="CountryBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="CountryContent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>211</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit#CountryContent
{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="ProjectDetail">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>250</width>
       <height>270</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame#ProjectDetail
{
	background-color: rgba(213, 195, 121, 210);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="ProjectDetailHint">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>130</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel#ProjectDetailHint
{
	background-color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="text">
       <string>  按照项目编号查询：</string>
      </property>
     </widget>
     <widget class="QComboBox" name="ProjectBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>60</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="ProjectContent">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>211</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit#ProjectContent
{
	background-color: rgb(255, 255, 255);
}
</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
